sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+  - A configuration profile to test the pipeline with a full-size test dataset
+
+:::note
+Some configuration files contain the definition of profiles that can be flexibly applied. For example, the `docker`, `singularity`, and `conda` profiles are defined in the `nextflow.config` file in the pipeline project directory.
+:::
+
+Profiles are sets of configuration options that can be flexibly applied to a pipeline.
+They are also commonly defined in the `nextflow.config` file in the root of the pipeline directory.
+
+Profiles used by nf-core pipelines can be broadly categorized into two groups:
+
+- Software management profiles
+  - Profiles for the management of software dependencies using container or environment management tools, for example, `docker`, `singularity`, and `conda`.
+- Test profiles
+  - Profiles to execute the pipeline with a standardized set of test data and parameters, for example, `test` and `test_full`.
+
+nf-core pipelines are required to define software containers and environments that can be activated using profiles. Although it is possible to run the pipelines with software installed by other methods (e.g., environment modules or manual installation), using container technology is more sharable, convenient, and reproducible.
+
+### Shared configuration files
+
+An `includeConfig` statement in the `nextflow.config` file is also used to include custom institutional profiles that have been submitted to the [nf-core config repository](https://github.com/nf-core/configs). At run time, nf-core pipelines will fetch these configuration profiles from the [nf-core config repository](https://github.com/nf-core/configs) and make them available.
+
+For shared resources such as an HPC cluster, you may consider developing a shared institutional profile.
+
+Follow [this tutorial](https://nf-co.re/docs/usage/tutorials/step_by_step_institutional_profile) to set up your own institutional profile.
+
+### Custom parameter and configuration files
+
+Nextflow will look for files that are external to the pipeline project directory. These files include:
+
+- The config file `$HOME/.nextflow/config`
+- A config file named `nextflow.config` in your current directory
+- Custom configuration files specified using the command line
+  - A parameter file that is provided using the `-params-file` option
+  - A config file that are provided using the `-c` option
+
+**Parameter file format**
+
+Parameter files are `.json` files that can contain an unlimited number of parameters:
+
+```json title="nf-params.json"
+{
+  "<parameter1_name>": 1,
+  "<parameter2_name>": "<string>",
+  "<parameter3_name>": true
+}
+```
+
+You can override default parameters by creating a `.json` file and passing it as a command-line argument using the `-param-file` option.
+
+```bash
+nextflow run nf-core/rnaseq -profile docker --input <path/to/input? --outdir <results> -param-file <path/to/nf-params.json>
+```
+
+**Configuration file format**
```

Comment:
If we are talking specifically about pipeline parameters, I would remove this entire section as this is not recommended by the Nextflow developers apparently - we should stick just to params json/yamls

Reply:
This section is more biased towards describing the file formats.

I'm lumping everything that is configuration realted together to mirror the nextflow docs. This get's complicated as there are params in the test config files as well. I'll add another warning elsewhere to highlight this again.